configs/CHONKERH735/config.h

```diff
@@ -123,8 +123,6 @@ Details: https://github.com/crteensy/yolo-chonker/tree/as-built-20230303
 
 #define DEFAULT_GYRO_TO_USE             GYRO_CONFIG_USE_GYRO_1
 #define GYRO_1_ALIGN                    CW0_DEG_FLIP
-#define GYRO_1_ALIGN_PITCH              1800
```

Comment:
(this one is correct - 0deg + flip)

Reply:
Would think to remove `#define GYRO_1_ALIGN                    CW0_DEG_FLIP` instead.